### ISSUE TITLE
Gravity Form Salesforce add-on not posting notice that the form was sucessfully added/updated to salesforce

### ISSUE THREAD

**Issue by zimmy55** [katzwebservices/Gravity-Forms-Salesforce#230]
![gform](https://cloud.githubusercontent.com/assets/13783441/9252849/c800eff0-4197-11e5-915e-b3ab8f470fa9.png)

**Comment by JackieRo** [comment]
I am seeing this issue as well right now!

**Comment by gmcinnes** [comment]
@zimmy55 @JackieRo When you say that it's not posting a notice, and you show that image, do you mean that the "Successfully added...." message is not appearing on the form entries? 

If that's the case, are the entries actually getting added to Salesforce?

**Comment by zimmy55** [comment]
The entries are getting into salesforce but there is no successfully added message.

Jeff

Jeffrey Evens
Web Specialist

[RGS Energy]<http://www.rgsenergy.com/>

833 W. South Boulder Road | Louisville, CO 80027
tel 303.222.8376 | mobile 678.878.7930 |

RGSEnergy.com<http://www.rgsenergy.com/> | Jeff.Evens@rgsenergy.com<mailto:jeff.evens@rgsenergy.com>

Confidentiality Note: This e-mail message may contain confidential or legally privileged information and is intended only for the use of the intended recipient(s). Any unauthorized disclosure, dissemination, distribution, copying or the taking of any action in reliance on the information herein is prohibited. E-mails are not secure and cannot be guaranteed to be error free as they can be intercepted, amended, or contain viruses. Anyone who communicates with us by e-mail is deemed to have accepted these risks. RGS Energy is not responsible for errors or omissions in this message and denies any responsibility for any damage arising from the use of e-mail. Any opinion and other statement contained in this message and any attachment are solely those of the author and do not necessarily represent those of the company.

**Comment by gmcinnes** [comment]
Hmm. Ok.

-g

**Comment by JackieRo** [comment]
Yeah I actually just created a new form and new integration and it worked. Not sure what was happening with the old form or why it wasn't working.

**Comment by zimmy55** [comment]
I’ll have to give it another try


Jeffrey Evens
Web Specialist

[RGS Energy]<http://www.rgsenergy.com/>

833 W. South Boulder Road | Louisville, CO 80027
tel 303.222.8376 | mobile 678.878.7930 |

RGSEnergy.com<http://www.rgsenergy.com/> | Jeff.Evens@rgsenergy.com<mailto:jeff.evens@rgsenergy.com>

Confidentiality Note: This e-mail message may contain confidential or legally privileged information and is intended only for the use of the intended recipient(s). Any unauthorized disclosure, dissemination, distribution, copying or the taking of any action in reliance on the information herein is prohibited. E-mails are not secure and cannot be guaranteed to be error free as they can be intercepted, amended, or contain viruses. Anyone who communicates with us by e-mail is deemed to have accepted these risks. RGS Energy is not responsible for errors or omissions in this message and denies any responsibility for any damage arising from the use of e-mail. Any opinion and other statement contained in this message and any attachment are solely those of the author and do not necessarily represent those of the company.

**Comment by JackieRo** [comment]
This is an issue again on the same form, it is no longer sending entries to Salesforce and it is not logging the entry notes so I cannot see what is wrong. Any ideas?
